Styled story question and answer elements

grodeska opened the pull request.

grodeska committed:
styled question and answer forms

grodeska committed:
styled story question and answers

jamestyack commented:
See a few conflicts on these files from master Lawrence .. any chance you could take a look?  @philril may be able to help... there seems to be overlap on some of the things being updated.

```
CONFLICT (content): Merge conflict in client/styles/_header.scss
CONFLICT (content): Merge conflict in client/styles/_card.scss
CONFLICT (content): Merge conflict in client/index.html
Automatic merge failed; fix conflicts and then commit the result.
```

philril commented:
@jamestyack @grodeska My bad, this was from some updates I made. I fixed the merge conflicts and it's good to go. (I did have to do it in a separate branch though. I incorporated all Lawrence's changes in this separate PR(#92): https://github.com/civicmakers/client/pull/92, which when merged will allow us to delete _this_ PR(#90))

grodeska commented:
Thanks, Phil! Much appreciated :)

On Sun, Dec 6, 2015 at 5:24 PM, Philip Riley notifications@github.com
wrote:

> @jamestyack https://github.com/jamestyack @grodeska
> https://github.com/grodeska My bad, this was from some updates I made.
> I fixed the merge conflicts and it's good to go. (I did have to do it in a
> separate branch though. I incorporated all Lawrence's changes in this
> separate PR(#92 https://github.com/civicmakers/client/pull/92): #92
> https://github.com/civicmakers/client/pull/92, which when merged will
> allow us to delete _this_ PR(#90
> https://github.com/civicmakers/client/pull/90))
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/civicmakers/client/pull/90#issuecomment-162385475.

## 

@grodeska http://twitter.com/grodeska
http://grodeska.com
415-819-8495
